nvm.sh

```diff
@@ -937,7 +936,11 @@ nvm_install_node_binary() {
 
   if [ -n "$NVM_OS" ]; then
     if nvm_binary_available "$VERSION"; then
-      t="$VERSION-$NVM_OS-$(nvm_get_arch)"
+      nvm_arch="$(nvm_get_arch)"
```

Comment:
Please use `local` here, and for `ksh` compatibility, define it in two lines, like so:
```
local NVM_ARCH
NVM_ARCH="$(nvm_get_arch)"
```